netfilterqueue/_impl.pyx

```diff
@@ -129,6 +142,46 @@ cdef class Packet:
 
         self._verdict_is_set = True
 
+    cpdef get_indev(self, bint name=False):
```

Comment:
I think this should be two functions, one to get the index and one to get the name. (Or an attribute for the index and a function for the name, which might make more sense especially if you're saving the indices on the object anyway like I suggested above.) Parameters that change the return type of a function are awkward; they'll almost certainly be passed as constants, and they create a hazard for calling code to not name the parameter resulting in difficult-to-understand things like `packet.get_indev(True)`.

Reply:
tbh, i dont use the name functionality. i added that to make it more complete as a lib.